app/views/hosts/_form.html.erb

```diff
@@ -27,8 +27,8 @@
     <div class="tab-content">
 
       <div class="tab-pane active" id="primary">
-        <%= text_f f, :name, :size => "col-md-5", :value => name_field(@host),
-            :help_inline => _("This value is used also as the host's primary interface name.") %>
+        <%= text_f f, :name, :size => "col-md-4", :value => name_field(@host),
+            :help_inline => link_to_function(icon_text("random", _("Randomize")), "randomizeName()") + ' ' + _("This value is used also as the host's primary interface name.") %>
```

Comment:
Done, also it nows preselect and focus the name on load and on click so it's more usable. Added tooltip as well that explains how to disable this feature.